kernel/src/memory/mod.rs

```diff
@@ -717,7 +717,7 @@ unsafe impl GlobalAlloc for KernelAllocator {
                             .release_large_page(frame)
                             .expect("Can't deallocate frame");
                     } else {
-                        error!("Loosing large memory region. Oh well.")
+                        log::debug!("Loosing large memory region. Oh well.")
```

Comment:
just curious why you changed it, does this happen a lot now in rackscale?

Reply:
I noticed that during the vmops unmaplat test I see it happening fairly often; on other tests, no

Reply:
Got it, are we unmapping something larger than 4K there?

Reply:
Ah, I'm very sorry. I should have looked at the line closer - I actually almost never see this print in the output of any of the tests. It's the print statement on line 645 (that I also changed to debug) that I was seeing in the unmaplat bench.

I figured since we were going to do actual benchmarks soon it would be prudent to remove print statements that happen during memory allocations; however, I'm happy to change this one back to an error since I almost never see it.